kii/kii_api_call.c

```diff
@@ -12,6 +12,7 @@ kii_code_t kii_api_call_start(
 {
     khc_reset_except_cb(&kii->_khc);
     _reset_buff(kii);
+    _req_headers_free_all(kii);
```

Comment:
I think this should come before `khc_reset_except_cb`

Reply:
I have changed to avoid confusion, but I don't think it makes a substantial change, as the `khc_reset_except_cb` sets `khc->_headers` to NULL and `_req_headers_free_all` deals with `kii->_headers`, not with `kii->_khc->_headers`.